Code/Framework/AzFramework/Tests/ArchiveCompressionTests.cpp

```diff
@@ -35,29 +35,6 @@ namespace UnitTest
             : m_application { AZStd::make_unique<AzFramework::Application>() }
         {}
 
-        void SetUp() override
-        {
-            AZ::SettingsRegistryInterface* registry = AZ::SettingsRegistry::Get();
-
-            auto projectPathKey =
-                AZ::SettingsRegistryInterface::FixedValueString(AZ::SettingsRegistryMergeUtils::BootstrapSettingsRootKey) + "/project_path";
-            AZ::IO::FixedMaxPath enginePath;
-            registry->Get(enginePath.Native(), AZ::SettingsRegistryMergeUtils::FilePathKey_EngineRootFolder);
-            registry->Set(projectPathKey, (enginePath / "AutomatedTesting").Native());
-            AZ::SettingsRegistryMergeUtils::MergeSettingsToRegistry_AddRuntimeFilePaths(*registry);
-
-            m_application->Start({});
-            // Without this, the user settings component would attempt to save on finalize/shutdown. Since the file is
-            // shared across the whole engine, if multiple tests are run in parallel, the saving could cause a crash 
-            // in the unit tests.
-            AZ::UserSettingsComponentRequestBus::Broadcast(&AZ::UserSettingsComponentRequests::DisableSaveOnFinalize);
-        }
-
-        void TearDown() override
-        {
-            m_application->Stop();
-        }
-
     private:
         AZStd::unique_ptr<AzFramework::Application> m_application;
```

Comment:
Should this be removed as well?

Reply:
The application object still needs to be constructed for this test to work, but not started. Though I'm sure all of these tests can be rewritten without depending directly on AzFramework::Application

Reply:
Maybe add a comment about that, otherwise it looks unused.

Reply:
The [AzFramework Application constructor](https://github.com/o3de/o3de/blob/development/Code/Framework/AzFramework/AzFramework/Application/Application.cpp#L116-L134) initializes the AZ::IO::Archive(CryPak) and the AZ::IO::FileIOBase to setup the FileIO Aliases, so it is not needed to invoke Start and Stop.
FYI the base class Component Application constructor initializes the Settings Registry so that is available as well.

Now the important part of the changes to note is that because we don't invoke `Application::Start` anymore, the [UserSettingsComponent](https://github.com/o3de/o3de/blob/development/Code/Framework/AzFramework/AzFramework/Application/Application.cpp#L338) is not created, therefore we don't have to worry about it trying to write to the `<project-path>/user/UserSettings.xml` when the `Application::Stop` is invoked.

TLDR: This change should be fine.